### DISCUSSION TITLE
Question : Does Spotify work ?

### DISCUSSION THREAD

**Discussion by d4rklynk** [Divested-Mobile/DivestOS-Build#110] [General] [answered]
I feel bad to ask that, but this the only app I can't make the switch.
Thanks in advance

**Comment by SkewedZeppelin** [comment, marked as answer]
Spotify was recently reported working by a user.
Which is actually surprising since Widevine DRM is removed.

**Reply by d4rklynk** [reply]
Thanks a lot, DivestOS it is then ^^

**Comment by d4rklynk** [comment]
I just finish instaling DOS, and Spotify works as expected.
- Downloads work (for premium account)
- Spotify Connect (remote connect/control other devices) seems to work
- Notification Panel with controls work
